ddtrace/contrib/asyncpg/patch.py

```diff
@@ -113,16 +115,20 @@ async def _traced_query(pin, method, query, args, kwargs):
 
         # set span.kind to the type of request being performed
         span.set_tag_str(SPAN_KIND, SpanKind.CLIENT)
-
         span.set_tag(SPAN_MEASURED_KEY)
         span.set_tags(pin.tags)
+
+        dbm_propagator = getattr(config.asyncpg, "_dbm_propagator", None)
```

Comment:
Using `core.dispatch()` from `ddtrace.internal.core` here would keep DBM's product logic separate from the instrumentation logic that this file focuses on.

Reply:
I've separated all contrib code from DBM code, though I used `core.dispatch_with_results`

Reply:
Thinking about this more, we can probably use just `core.dispatch` tho, since the objects being changed (`span`, `args`, and `kwargs`) are all mutable objects, correct?